components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -1283,10 +1295,10 @@ bool PublisherInfoDatabase::MigrateV5toV6() {
 
     const std::string columns_select = "publisher_id, "
                                        "sum(duration) as duration, "
-                                       "sum(visits) as duration, "
+                                       "sum(visits) as visits, "
                                        "sum(score) as score, "
-                                       "percent, "
-                                       "weight, "
+                                       "sum(percent) as percent, "
+                                       "sum(weight) as weight, "
```

Comment:
did these 3 changes fix a bug?

Reply:
yes, but it wouldn't really effect users because as soon as they would visit a site, this values are re-calculated